packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -60,11 +60,19 @@ export const DefaultKaizenSiteDemo: ComponentStory<
           <>
             <FilterMultiSelect.SearchInput />
             <FilterMultiSelect.ListBox>
-              {({ allItems }) =>
-                allItems.map(item => (
+              {({ allItems, hasNoItems }) => {
+                if (hasNoItems) {
+                  return (
+                    <FilterMultiSelect.NoResults>
+                      No results found.
```

Comment:
Unfortunately I wasn't able to add the search term to this message for the non-async examples because the state of the search field is all uncontrolled and we don't have access to it here.

If we want to tackle this, it's going to be a separate piece of work.